lib/manifold.ex

```diff
@@ -1,42 +1,63 @@
 defmodule Manifold do
   use Application
 
-  alias Manifold.{Partitioner, Utils}
+  alias Manifold.Partitioner
+  alias Manifold.Sender
+  alias Manifold.Utils
 
   @max_partitioners 32
   @partitioners min(Application.get_env(:manifold, :partitioners, 1), @max_partitioners)
   @workers_per_partitioner Application.get_env(:manifold, :workers_per_partitioner, System.schedulers_online)
 
+  @max_senders 128
+  @senders min(Application.get_env(:manifold, :senders, 1), @max_senders)
```

Comment:
It might make more sense here to treat this like workers_per_partition and default it to running System.schedulers_online but allow an override.

Reply:
I went back and forth on this. Mainly didn't want to start a bunch of processes if the consumer of this library didn't intend to use the offload send mode.

So I settled for documenting it in the README. I could see the danger where someone just used it without reading the manual.

In that sense I agree that setting it to `System.schedulers_online` is safer. Maybe since erlang processes are so cheap, the "safe by default" route is better?

Reply:
Yea, I think the best efficiency for the sender is to have one per scheduler, and even if you had a super beefy machine with 100 schedulers, the 100 processes aren't going to cause you any heartburn.

Reply:
Such luxury that is the BEAM...